ArduPlane/Parameters.cpp

```diff
@@ -1184,6 +1184,13 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("RUDD_DT_GAIN", 9, ParametersG2, rudd_dt_gain, 10),
 
+    // @Param: MANUAL_RCMASK
+    // @DisplayName: Manual R/C pass-through mask
+    // @Description: Mask of R/C channels to pass directly to corresponding output channel when in MANUAL mode. When in any more except MANUAL the channels selected with this option behave normally. This parameter is designed to allow for complex mixing strategies to be used for MANUAL flight using transmitter based mixing. Note that when this option is used you need to be very careful with pre-flight checks to ensure that the output is correct both in MANUAL and non-MANUAL modes.
```

Comment:
When in any more except MANUAL -> When in any **mode** except MANUAL

Right? I was confused reading it.